### PR TITLE
UX: Allow pasting lat / lng in a single operation

### PR DESCRIPTION
**Pull request by reubendowle**
This speeds up the workflow when pasting location found in external mapping application

Acceptance Criteria:

- [ ] Features and enhancements must be fully implemented so that they can be released at any time without additional work
- [ ] Automated unit and/or acceptance tests are mandatory to ensure the changes work as expected and to reduce repetitive manual work
- [ ] Frontend components must be responsive to work and look properly on phones, tablets, and desktop computers; you must have tested them on all major browsers and different devices
- [ ] Documentation and translation updates should be provided if needed
- [ ] In case you submit database-related changes, they must be tested and compatible with SQLite 3 and MariaDB 10.5.12+

### CODE REVIEW

File: frontend/src/dialog/photo/edit/details.vue [hunk 1]
```diff
@@ -469,6 +471,27 @@ export default {
 
       this.time = taken.toFormat("HH:mm:ss");
     },
+    pastePosition(event) {
+      /* If text is pasted in the form of "XXX.XXXX, YYY.YYYY" (for example if
+       * pasting a position from external mapping app) then auto-fill both lat
+       * and lng fields
+       */
+      let paste = (event.clipboardData || window.clipboardData).getData("text");
+      let s = paste.split(/[ ,]+/);
```
Comment (lastzero) [inline frontend/src/dialog/photo/edit/details.vue:480]: Looks like it might throw an exception if paste isn't a string for some reason?

**Comment by lastzero** [comment]
Thank you very much! I didn't know a `@paste` event exists and can be used with VueJS. Will this work with all browsers? Does it require further testing e.g. with browsers you haven't tested yet?

**Comment by lastzero** [comment]
As I'm not sure how much time I have for this next week, I'll move ahead to merge your PR and add checks in case the browser clipboard cannot be accessed for some reason. Thank you for your contribution! :heart:

**Comment by reubendowle** [comment]
> Thank you very much! I didn't know a @paste event exists and can be used with VueJS. Will this work with all browsers? Does it require further testing e.g. with browsers you haven't tested yet?

It seems you can capture any DOM event in Vue. This should work with basically any browser. They all support the paste event (https://developer.mozilla.org/en-US/docs/Web/API/Element/paste_event#browser_compatibility)

File: frontend/src/dialog/photo/edit/details.vue (hunk 1 above)
Comment (reubendowle) [reply frontend/src/dialog/photo/edit/details.vue:480]: The specification of this seems to indicate that you will always get an empty string result even if the data type does not support strings (https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer/getData#return_value).

I tested this by pasting some non-text data (eg image data), and the code just ends up with s.length == 0, and thus ignoring it.

**Pull request merged**